src/viur/core/modules/moduleconf.py

```diff
@@ -68,16 +68,36 @@ def get_by_module_name(cls, module_name: str) -> None | SkeletonInstance:
 @StartupTask
 def read_all_modules():
     db_module_names = [m["name"] for m in db.Query("viur-module-conf").run(999)]
```

Comment:
Please make the `"viur-module-conf"` more dynamic. Either by using a skeleton.kindName (which is not the case here - maybe use `skel_cls = self._resolveSkelCls()` and use `skel_cls.kindName`, or using a constant defined on top.)

Reply:
But can the `kindName` change ? Because we set the module here hardcoded.
https://github.com/viur-framework/viur-core/blob/486d500c0e226feac67c28bdd3d2a6d0f27c4fb9/src/viur/core/__init__.py#L123
https://github.com/viur-framework/viur-core/blob/486d500c0e226feac67c28bdd3d2a6d0f27c4fb9/src/viur/core/__init__.py#L127)